device-types/Cisco/MDS-C9148-16P.yaml

```diff
@@ -0,0 +1,120 @@
+---
+manufacturer: Cisco
+model: MDS 9148 16P
+slug: ds-9148-16p
+part_number: DS-C9148-16P-K9
+u_height: 1
+is_full_depth: true
```

Comment:
Please add a comment in markdown format with a link to the manufacturers datasheet, if you are able to find a publicly available one